docs/pages/access-controls/access-graph/self-hosted-helm.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Run Teleport Access Graph on Self-Hosted Clusters with Helm
+---
+
+Using Access Graph with a self-hosted Teleport cluster requires
+setting up the Teleport Access Graph (TAG) service.
+TAG is a dedicated service which uses PostgreSQL as its backing storage
+and communicates with Auth Service and Proxy Service
+to collect information about resources and access.
+
+This guide will help you set up the TAG service
+in a Kubernetes cluster using a Helm Chart,
+and enable the Access Graph feature in your Teleport cluster.
+
+The full listing of supported parameters can be found in the [`teleport-access-graph` Helm chart reference](../../reference/helm-reference/teleport-access-graph.mdx)
+
+Teleport Access Graph is enabled by default for Teleport Team users.
+You can skip the setup process here and start using Access Graph right away
+by signing up for a [Teleport Team trial](https://goteleport.com/signup/).
+
+## Prerequisites
+
+- Kubernetes >= v1.21
+- Helm >= (=helm.version=)
+- A running Teleport Enterprise cluster v14.3.4 or later.
+  - For the purposes of this guide, we assume that the Teleport cluster is set up
+    [using a Helm chart](../../deploy-a-cluster/helm-deployments.mdx)
+    in the same Kubernetes cluster that will be used to deploy Teleport Access Graph .
+- An updated `license.pem` with Teleport Policy enabled.
+- A PostgreSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to the database with needs to be the owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+  - Amazon RDS for PostgreSQL is supported.
+- A TLS certificate for the Access Graph service
```

Comment:
Might be good to clarify that you can also use a tool like cert-manager to issue certificates automatically (i.e., we don't just require a certificate file).

Reply:
I don't think we support cert reloading, so cert-manager will renew certs but TAG will not pick them up.